CleverPush/Source/CleverPushInstance.m

```diff
@@ -1081,6 +1081,15 @@ - (void)handleDidFailRegisterForRemoteNotification:(NSError*)err {
 
 #pragma mark - register Device Token
 - (void)registerDeviceToken:(id)newDeviceToken onSuccess:(CPResultSuccessBlock)successBlock onFailure:(CPFailureBlock)failureBlock {
+    if([[NSUserDefaults standardUserDefaults] objectForKey:CLEVERPUSH_DEVICE_TOKEN_KEY] != nil) {
```

Comment:
Code style: space after `if ` missing
2 if's can be combined to one if

Reply:
Also we can simplify by puting the conditions below here:
https://github.com/cleverpush/cleverpush-ios-sdk/pull/88/files#diff-f5326f84792597a7145e9b2f5be8823e3bc651f5b4faac02dc05dca951eee90fR1121

Reply:
Because of the early-exits, at the bottom of this function we are sure that new != old device token.

Reply:
> Also we can simplify by puting the conditions below here: https://github.com/cleverpush/cleverpush-ios-sdk/pull/88/files#diff-f5326f84792597a7145e9b2f5be8823e3bc651f5b4faac02dc05dca951eee90fR1121

Hello @gbhrdt Can you please give the correct link?

Reply:
@cpandya25  Line 1121

![image](https://user-images.githubusercontent.com/1145196/196167630-f8c96017-87d4-4452-842d-1a76168c9736.png)

Reply:
@gbhrdt For the confirmation, I am asking that we need to put this code.

Reply:
Yes, but then we probably dont need the condition. Please check the code flow.

Reply:
@gbhrdt Okay, let me check how the token changes work another time with restoring the device.

Reply:
Before taking backup 
![Screenshot 2022-10-18 at 10 54 03 AM](https://user-images.githubusercontent.com/110810122/196345452-c610c58b-0b15-4199-b737-a196ff1579e7.png)

Reply:
After taking backup
![Screenshot 2022-10-18 at 11 08 31 AM](https://user-images.githubusercontent.com/110810122/196345528-9744f1ee-4998-4155-9df1-fe5d41b8bcf0.png)